jobs/JEVS_MESOSCALE_PLOTS

```diff
@@ -99,19 +99,20 @@ fi
 ####################################
 # Define COMIN/COMOUT variables:
 ####################################
-export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
-export COMINapcp24mean=${COMINapcp24mean:-$(compath.py $envir/com/$NET/$evs_ver)/stats/$COMPONENT}
+export EVSINapcp24mean=${EVSINapcp24mean:-$COMIN/stats/$COMPONENT}
+export EVSINccpa=${EVSINccpa:-$COMIN/prep/cam/$RUN}
+export EVSINmrms=${EVSINmrms:-$COMIN/prep/cam/$RUN}
+export EVSINspcotlk=${EVSINspcotlk:-$COMIN/prep/cam/$RUN}
+export cartopyDataDir=${cartopyDataDir:-/apps/ops/prod/data/cartopy}
```

Comment:
We will need to change this line. But I am not sure how yet. NCO will not allow hardcoded paths. I have asked @MarcelCaron-NOAA how we can work around this hardcoded path.

Reply:
This line?
export cartopyDataDir=${cartopyDataDir:-/apps/ops/prod/data/cartopy}

I had noticed it before, but thought it was approved.

Reply:
> This line? export cartopyDataDir=${cartopyDataDir:-/apps/ops/prod/data/cartopy}
> 
> I had noticed it before, but thought it was approved, because there is no module for this as far as I know.

Correct, the cartopyDataDir line. Mesoscale is not the only component using it. We will see how to address this one. If it is not anything straight forward, we will note it to be addressed later and merge this PR.